packages/experiments/src/components/CollapsibleSection/CollapsibleSection.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { ICollapsibleSectionProps, ICollapsibleSectionStyles } from './CollapsibleSection.types';
+import { ITheme } from 'office-ui-fabric-react';
+
+export const getStyles = (props: ICollapsibleSectionProps & { theme: ITheme }): ICollapsibleSectionStyles => {
+  const { theme } = props;
+
+  return {
+    child: [
+      {
+        paddingLeft: 30,
```

Comment:
is this important enough to be a prop, or maybe some future theme value?

Reply:
As I add examples (todo) this will get hammered out